Resources/AzureDevOps/JobTemplates/Deploy.yml

```diff
@@ -0,0 +1,110 @@
+parameters:
+  AzureSubscription: ''
+  AksAdClientApplicationName: ''
+  AksAdServerApplicationName: ''
+  AksServicePrincipalName: ''
+  ApimApisToRemove: '"Echo API"' #Comma seperated list of strings wrapped in double quotes
+  CertificateIssuerAccountId: ''
+  CertificateIssuerAdministratorPhoneNumber: ''
+  CertificateIssuerPassword: ''
+  DfcDevOpsScriptRoot: ''
+  envAbbreviation: ''
+  Environment: ''
+  EnvironmentTag: ''
+  ResourceGroup: ''
+  ParentBusinessTag: ''
+  ServiceOfferingTag: ''
+  SharedKeyVaultName: ''
+  SharedPrefix: ''
+  StorageAccountName: ''
+  
+jobs:
+- deployment: DeployTo_${{ parameters.Environment }}
+  pool:
```

Comment:
Suggestion: I'd be tempted to pull this out to the top level, so it affects everything

Reply:
Moved it